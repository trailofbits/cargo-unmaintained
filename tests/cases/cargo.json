[
  {
    "name": "adler",
    "outdated_deps": [],
    "repo_status": "Archived",
    "version": "1.0.2"
  },
  {
    "name": "im-rc",
    "outdated_deps": [
      {
        "name": "bitmaps",
        "req": "^2",
        "version_latest": null,
        "version_used": "2.1.0"
      },
      {
        "name": "sized-chunks",
        "req": "^0.6.4",
        "version_latest": null,
        "version_used": "0.6.5"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "15.1.0"
  },
  {
    "name": "miow",
    "outdated_deps": [
      {
        "name": "windows-sys",
        "req": "^0.48.0",
        "version_latest": null,
        "version_used": "0.48.0"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.6.0"
  },
  {
    "name": "oorandom",
    "outdated_deps": [],
    "repo_status": "Uncloneable",
    "version": "11.1.3"
  },
  {
    "name": "partial_ref_derive",
    "outdated_deps": [
      {
        "name": "syn",
        "req": "^1.0.40",
        "version_latest": null,
        "version_used": "1.0.109"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.3.3"
  },
  {
    "name": "rusty-fork",
    "outdated_deps": [
      {
        "name": "quick-error",
        "req": "^1.2",
        "version_latest": null,
        "version_used": "1.2.3"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.3.0"
  },
  {
    "name": "serde-value",
    "outdated_deps": [
      {
        "name": "ordered-float",
        "req": "^2.0.0",
        "version_latest": null,
        "version_used": "2.10.1"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.7.0"
  },
  {
    "name": "utf8parse",
    "outdated_deps": [],
    "repo_status": "Unassociated",
    "version": "0.2.1"
  },
  {
    "name": "varisat-dimacs",
    "outdated_deps": [
      {
        "name": "itoa",
        "req": "^0.4.4",
        "version_latest": null,
        "version_used": "0.4.8"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.2.2"
  },
  {
    "name": "varisat-internal-macros",
    "outdated_deps": [
      {
        "name": "syn",
        "req": "^1.0.40",
        "version_latest": null,
        "version_used": "1.0.109"
      },
      {
        "name": "synstructure",
        "req": "^0.12.4",
        "version_latest": null,
        "version_used": "0.12.6"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.2.2"
  },
  {
    "name": "varisat",
    "outdated_deps": [
      {
        "name": "itoa",
        "req": "^0.4.4",
        "version_latest": null,
        "version_used": "0.4.8"
      },
      {
        "name": "ordered-float",
        "req": "^2.0.0",
        "version_latest": null,
        "version_used": "2.10.1"
      },
      {
        "name": "vec_mut_scan",
        "req": "^0.3.0",
        "version_latest": null,
        "version_used": "0.3.0"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.2.2"
  }
]
