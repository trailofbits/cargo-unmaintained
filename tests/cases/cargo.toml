url = "https://github.com/rust-lang/cargo"
rev = "0.77.0"
use_real_github = true
