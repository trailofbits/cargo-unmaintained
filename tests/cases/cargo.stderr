Scanning 341 packages and their dependencies (pass --verbose for more information)
warning: failed to determine `oorandom` existence: unexpected response code: 502
warning: failed to clone `https://sr.ht/~icefox/oorandom/`: [
    "fatal: remote error: /n/nYou have tried to clone a project from sourcehut, but you probably meant to/nclone a specific git repository for this project instead. A single project on/nsourcehut often has more than one git repository./n/n/n/nTo browse all of the available repositories for this project, visit this URL:/n/n  https://sr.ht/~icefox/oorandom/sources/n",
]
warning: failed to clone `https://hg.sr.ht/~icefox/oorandom`: [
    "fatal: repository 'https://hg.sr.ht/~icefox/oorandom/' not found/n",
]
